client/pages/login.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import styles from '../styles/login.module.css';
+
```

Comment:
We need to think about a welcome page @alexandra-plana @lexarisa so we can place sign in and sign up in this welcome page